pkg/scheduler/plugins/deviceshare/plugin.go

```diff
@@ -545,12 +546,43 @@ func (p *Plugin) preBindObject(ctx context.Context, cycleState *framework.CycleS
 		return nil
 	}
 
+	var deviceAllocs *apiext.DeviceAllocations
```

Comment:
can we consider add move this logic to Reserve() phase?

Reply:
ok